cns/service/main.go

```diff
@@ -409,7 +409,7 @@ func sendRegisterNodeRequest(httpc *http.Client, httpRestService cns.HTTPService
 	}
 	defer response.Body.Close()
 
-	if response.StatusCode != http.StatusCreated {
+	if response.StatusCode != http.StatusOK {
```

Comment:
are there no unit tests for this? if not, some should be added to lock down the expected responses.

Reply:
@smittal22 can you add a test for this to prevent the behavior from drifting accidentally?

Reply:
Added the UTs.